CHANGELOG.md

```diff
@@ -1,25 +1,58 @@
+[//]: # "This is how you write comments in markdown."
+
 Change Log
-====================
+==========
 
 Introduction
 ------------
-Drake is undergoing continuous development and cannot yet offer a stable, documented API or ABI. To help downstream users cope, this document provides brief summaries of the changes. The changes are organized by pull requests that are merged into Drake's master branch.
 
-Descriptions of Pull Requests
-------------------------------
+Drake is undergoing continuous development and cannot yet offer a
+stable API.  To help downstream users cope, this document provides
+brief summaries of user-facing changes.
+
+Unreleased: changes on master, not yet released
+-----------------------------------------------
+
+[//]: # "New functionality or APIs."
+### Added
+
+ - None
+
+[//]: # "Altered functionality or APIs."
+### Changed
+
+ - [#1992][] Matlab tests must now be explicitly listed in CMakeLists.
+ - [#1970][] The drake pod Makefile now includes the `install` action.
+ - [#1953][] Replace valuecheckMatrix() with CompareMatrices().
+ 
+[//]: # "Lost functionality or APIs."
+### Removed / Deprecated
 
-### ([@liangfok][], [#1953][]) - Replacement of valuecheckMatrix() with CompareMatrices()
+ - None
 
-Method `valuecheckMatrix()` in `drake/util/testUtil.h` was removed and replaced with `drake::util::CompareMatrices()` in `drake/util/eigen_matrix_compare.h`.
+[//]: # "Smaller bug fixes.  No API changes."
+### Fixes
 
-Unit tests that originally contained
+ - [#2008][] `Drake::simulate(4-arg)` actually compiles again now.
+ - [#1990][] Windows builds that include bullet are working again.
```

Comment:
I think the last three are probably all too small to inckude in the changelog, no?